libcst/_parser/_parso/_pgen2/_generator.py

```diff
@@ -0,0 +1,361 @@
+# A fork of `parso.pgen2.generator`.
```

Comment:
Regarding this file name:

I originally only intended for one underscore in each module's fully qualified name as a way of marking certain modules as private.

When we ended up reworking some of the internal package structure, we ended up with multiple underscores in package names.

I get that this is a matter of opinion, but based on my original idea for underscores, that would mean that this module should be `libcst._parser.parso.pgen2.generator`, not `libcst._parser._parso._pgen2._generator`.

It's up to you what pattern you want to follow.

Reply:
Hmm, I wasn't really sure what our methodology was here, since it wasn't consistent to start with. That seems like a relatively easy change to make, I'll put it on the  backlog to fix all of the relevant areas both within parso and elsewhere.